Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -5531,6 +5531,8 @@
 		E7E0357224D4E196008DFEFB /* AnalyserOptions.h in Headers */ = {isa = PBXBuildFile; fileRef = E7E0357124D4E191008DFEFB /* AnalyserOptions.h */; };
 		E7FBE7C224EB2FB000A18E62 /* StereoPannerNode.h in Headers */ = {isa = PBXBuildFile; fileRef = E7FBE7C124EB2FAB00A18E62 /* StereoPannerNode.h */; };
 		E7FBE7C324EB2FB600A18E62 /* StereoPannerOptions.h in Headers */ = {isa = PBXBuildFile; fileRef = E7FBE7C024EB2FAB00A18E62 /* StereoPannerOptions.h */; };
+		E8FC921B285BFD5B004904B2 /* JSWindowEventHandlers+Gamepad.h in Product Dependencies */ = {isa = PBXBuildFile; fileRef = E8FC9219285BFD5A004904B2 /* JSWindowEventHandlers+Gamepad.h */; };
+		E8FC921C285BFD5B004904B2 /* JSWindowEventHandlers+Gamepad.cpp in Product Dependencies */ = {isa = PBXBuildFile; fileRef = E8FC921A285BFD5B004904B2 /* JSWindowEventHandlers+Gamepad.cpp */; };
```

Comment:
I dont think we want this in the `.xcodeproj` as it kinda negates us adding it to `Source/WebCore/Sources.txt`.  You should be able to uncheck the WebCore target in the details sidebar or manually edit this file to remove this line.